scripts/build.js

```diff
@@ -40,6 +40,7 @@ class Filter extends Transform {
 
 // Colors normally don't work when using spawn(), so here we re-enable colors.
 process.env.FORCE_COLOR = require('supports-color').stdout.level;
+process.env.REACT_APP_VERSION = require(__dirname + "/../package.json").version
```

Comment:
Github won't let me comment on this, but up on line 20, the URL needs to be updated to:
```
  https://cra.link/deployment
```